DbMocker/ColumnSequenceContainer.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+
+namespace Apps72.Dev.Data.DbMocker
+{
+    /// <summary>
+    /// Container for configuring columns sequence
```

Comment:
Can you more explain the goal of this class?

Reply:
1. We use external types and don't assurance what describe fields and property didn't change order
2. We use some type for different queries with different column set and different column order
3. We want to set several identical names for the columns of one table. It needs for MultiMapping (https://dapper-tutorial.net/result-multi-mapping) when the primary keys of our tables are named the same

Reply:
@dvoituron, what do You think about configuring columns order?